### PR TITLE
Peer review- Validation Assist

### PR DESCRIPTION
**Pull request by Karuna-Mendix**

### CODE REVIEW

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags: ["studio pro", "mendix assist", "validation assist", "bot", "validation bo
 
 ## 1 Introduction 
 
-The Validation Assist helps you build validation microflows in a more automated way using pre-built expressions.  [Validation rules](/refguide/validation-rules/) are conditions that should be satisfied before an object is committed. There are different ways to [set up data validations](/refguide/setting-up-data-validation/) in Mendix. Building data validations in microflows is a manual and repetitive task and Validation Assist helps automate it. 
+The Validation Assist helps you build validation microflows in a more automated way using pre-built expressions. [Validation rules](/refguide/validation-rules/) are conditions that should be satisfied before an object is committed. There are different ways to [set up data validations](/refguide/setting-up-data-validation/) in Mendix. Building data validations in microflows is a manual and repetitive task and Validation Assist helps automate it. 
 
 The main features of the Validation Assist are the following:
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I would say - The main features of the Validation Assist are as follows:

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags: ["studio pro", "mendix assist", "validation assist", "bot", "validation bo
...
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md:14]: I found this sentence bit awkward. What about this - List of checks for all members whose data type can be empty, such as when a customer's name is not filled in a form

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 3]
```diff
@@ -22,10 +22,10 @@ The main features of the Validation Assist are the following:
 
 The Validation Assist is activated by default in the page elements where data validation might be needed. These elements are the following:
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I would suggest - These elements include:

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 4]
```diff
@@ -42,18 +42,18 @@ To access the Validation Assist, right-click a button or an input widget listed
...
 
-The **Generate Validation Microflow** dialog box is where you can configure all validations:
+The **Generate Validation Microflow** dialog box allows you to configure all validations:
 
 {{< figure src="/attachments/refguide/modeling/mx-assist-studio-pro/mx-validation-assist/generate-validation-microflow-dialog.png" alt="Generate Validation Microflow Dialog Box" >}}
 
 The dialog box consists of two sections:
 
-* **Microflow Properties,** where the general properties of the validation microflow are configured. Properties contain the following:
+* **Microflow Properties,** where the general properties of the validation microflow are configured. These properties include:
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md:52]: Capitalizing the first word in the list is just a suggestion. You can discard the changes I made here as you have maintained the consistency in the document.

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 5]
```diff
@@ -71,20 +71,20 @@ You can configure a custom **Save** button with the help of the Validation Assis
...
-* the name of the customer is valid (the name has only alphabetical symbols)
+* One or several fields in the form are not filled in
+* The selected date and time is in the future and it is within the opening hours (from 8 AM till 5 PM and not on a weekend)
+* The name of the customer is valid (the name has only alphabetical symbols)
 
 To configure the **Save** button for the use case above, do the following:
 
 1. Open the page with the form called *Appointment_NewEdit* and right-click the **Save** button in it.
-2. In the drop-down list, select **Generate validation microflow**:
+2. In the drop-down list, select **Generate validation microflow**.
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I am not fully convinced about using colon before the images in the steps. Please check.

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 6]
```diff
@@ -106,7 +106,7 @@ To configure the **Save** button for the use case above, do the following:
 
 6. You would like to save validations for the **Customer_Name** attribute (that the name is not empty and that it is alphabetical) as a sub-microflow to use them as a separate validation on a different page.  Do the following:
```
Comment (Karuna-Mendix) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I will suggest something like this - You would like to save validations for the **Customer_Name** attribute (ensuring that name is not empty and consist of alphabets only) as a sub-microflow to use them as a separate validation on a different page.

**Comment by Karuna-Mendix** [comment]
Hi @Yiyun333, I finished the review and made some changes. Let me know if you would like to discuss anything?

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md [hunk 7]
```diff
@@ -42,18 +42,18 @@ To access the Validation Assist, right-click a button or an input widget listed
...
 The dialog box consists of two sections:
 
-* **Microflow Properties,** where the general properties of the validation microflow are configured. Properties contain the following:
+* **Microflow Properties** where the general properties of the validation microflow are configured. These properties include:
 
     * **Microflow** – microflow name
     * **Commit** – defines whether changes should be committed to the database (only for buttons)
     * **Close Page** – defines whether the page should be closed after clicking the button (only for buttons)
-    * **On Event** – defines when to trigger the validation check (only for input widgets), for more information on events, see [On Click Events and Events Section](/refguide/on-click-event/)
+    * **On Event** – defines when to trigger the validation check (only for input widgets); for more information on events, see [On Click Events and Events Section](/refguide/on-click-event/)
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md:56]: I reverted the capitalization of the first word after the en dash. Those are not complete sentences, so I don't think we need to capitalize those first words there. See style guide: https://mendix.atlassian.net/wiki/spaces/RNDHB/pages/2520678744/Grammar+Formatting#Grammar-%26-Punctuation-with-Bulleted-Lists.

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md (hunk 5 above)
Comment (Yiyun333) [reply content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I haven't found anything about this in our style guide. Maybe we can discuss this during the group meeting next week? I personally like to use colon here. I found that connects the step and the image better. But using period could help to keep consistency in the list.

Comment (Karuna-Mendix) [reply content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: I didn't find anything relevant in the style guide either. Will discuss in the group meeting. I don't find anything wrong here, but this was the first instance I found. Let's see the outcome of the discussion.

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md (hunk 7 above)
Comment (Karuna-Mendix) [reply content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md:56]: I agree :)

File: content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md (hunk 5 above)
Comment (Yiyun333) [reply content/en/docs/refguide/modeling/mx-assist-studio-pro/mx-validation-assist.md]: @Karuna-Mendix Based on our group discussion, I think we've reached an agreement there that when the image directly follows one step/instruction, we can use colon there to keep the connection. I've made the changes and chosen colon over period in such cases :)

**Pull request merged**